tailwind.config.js

```diff
@@ -1,8 +1,35 @@
+import colors from "tailwindcss/colors";
+
 /** @type {import('tailwindcss').Config} */
 export default {
   theme: {
     extend: {
-      darkMode: true
+      darkMode: true,
+      colors: {
+        primary: {
+            ...colors.green
+        },
+        secondary: {
+          ...colors.red
+        },
+        neutral: {
+            '0': colors.white,
+            ...colors.gray,
+            '1000': colors.black,
+        },
+        error: {
+          ...colors.red
+        },
+        'text-lightMode': colors.slate['950'],
+        'text-darkMode': colors.slate['50'],
+        'background-lightMode': colors.gray['100'],
```

Comment:
ho visto che hai definito background-light e background dark, ma in giro per i componenti utilizzi bg-neutral-50. Usa queste due classi che hai definito.

fai diventare background --> bg